packages/chord-detect/index.ts

```diff
@@ -33,12 +37,41 @@ export function detect(source: string[]): string[] {
     .map((chord) => chord.name);
 }
 
+// Assumes that chord is presorted
+function findRoot(chord: string[]): string {
```

Comment:
Can you explain me _why_ this works? 🙏

Reply:
This is shamelessly borrowed from [this comment](https://music.stackexchange.com/a/17572), which I think explains it better than I could.

Unfortunately this will only work on chords forming major or minor triads.